internal/commands/app/create_test.go

```diff
@@ -490,26 +484,21 @@ func TestAppCreateHandler(t *testing.T) {
 					Values: []map[string]interface{}{},
 				}}}, appLocal.AppData)
 
-				// TODO(REALMC-8262): Investigate file path display options
-				dirLength := len(appLocal.RootDir)
-				fmtStr := fmt.Sprintf("%%-%ds", dirLength)
-
-				assert.Equal(t, strings.Join([]string{
-					"Successfully created app",
-					fmt.Sprintf("  Info             "+fmtStr, "Details"),
-					"  ---------------  " + strings.Repeat("-", dirLength),
-					fmt.Sprintf("  Client App ID    "+fmtStr, "remote-app-abcde"),
-					"  Realm Directory  " + appLocal.RootDir,
-					fmt.Sprintf("  Realm UI         "+fmtStr, "http://localhost:8080/groups/"+tc.expectedGroupID+"/apps/456/dashboard"),
-					"Check out your app: cd ./remote-app && realm-cli app describe",
-					"",
-				}, "\n"), out.String())
+				assert.Equal(t, fmt.Sprintf(`Successfully created app
+{
+  "client_app_id": "remote-app-abcde",
+  "filepath": %q,
+  "url": "http://localhost:8080/groups/%s/apps/456/dashboard"
+}
+Check out your app: cd ./remote-app && realm-cli app describe
+`, appLocal.RootDir, tc.expectedGroupID), out.String())
 
 				out.Reset()
 			})
 		}
 	})
 
+	// FIXME: these tests are incorrect, but waiting on REALMC-8830 to merge before truly fixing these
```

Comment:
Can we remove this comment now since REALMC-8839 has been merged?

Reply:
yes we can... thanks for catching!